lib/pbench/server/api/resources/query_apis/datasets/datasets_mappings.py

```diff
@@ -118,6 +115,7 @@ def _get(self, json_data: JSON, request: Request) -> Response:
             return jsonify(result)
         except TemplateNotFound:
             self.logger.exception(
-                "Document template {} not found in the database.", index_name
+                "Document template for index {} not found in the database",
+                index["index"],
             )
```

Comment:
It's too bad that we have to compose our own message text here instead of just using the `str()` of the `TemplateNotFound` exception object....

Reply:
Yeah that's a good point actually.